src/mixins/filing-template-mixin.ts

```diff
@@ -146,7 +147,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         correctedFilingId: this.getCorrectedFilingId,
         correctedFilingType: this.getCorrectedFilingType,
         correctedFilingDate: this.getCurrentDate,
-        comment: `${this.defaultCorrectionDetailComment}\n${this.getDetailComment}`
+        comment: `${this.defaultCorrectionDetailComment}\n${this.getDetailComment}`,
+        type: this.getClientErrorCorrection
```

Comment:
@severinbeauvais @lambert-alex This `type` will take one of two values "CLIENT/STAFF" when Edit UI fetches data from the backend.

Reply:
Yes. (Or null.)

PS You don't have to `@` me as I already receive a notification when a comment is added.